test/interface/element.js

```diff
@@ -17,4 +17,11 @@ div.innerHTML = '<span></span>'
 div.firstChild.outerHTML = '<p>hello</p> world';
 assert(div.toString(), '<div><p>hello</p> world</div>');
 
-assert(div.namespaceURI, 'http://www.w3.org/1999/xhtml');
\ No newline at end of file
+assert(div.namespaceURI, 'http://www.w3.org/1999/xhtml');
+
+const parser = new DOMParser();
+const doc = parser.parseFromString(`<hierarchy><android.view.View content-desc="text3&amp;more"/></hierarchy>`, 'text/xml').documentElement;
+
+assert(doc.firstChild.getAttribute('content-desc'), 'text3&amp;more');
+assert(doc.firstChild.outerHTML, '<android.view.View content-desc="text3&amp;more" />');
+assert(doc.innerHTML, '<android.view.View content-desc="text3&amp;more" />');
```

Comment:
I think we should add another test with the same content but for `text/html` case otherwise code-coverage is not happy

Reply:
I think `emptyAttributes` shouldn't apply to XML so I changed it a bit